<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>writeform</class>
 <widget class="QWidget" name="writeform">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,3,0">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="title">
      <string>مشخصات</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>شماره ورودی</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="number"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>تاریخ</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="date"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>متن</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QTextEdit" name="textEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="cancelBut">
       <property name="text">
        <string>انصراف</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="okBut">
       <property name="text">
        <string>تایید</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextBut">
       <property name="text">
        <string>بعدی</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
